docs/docs/policies/index.md

```diff
@@ -47,6 +49,17 @@ While specifying event filters is optional, policies must have the `name`, `desc
 
 More information about defining a scope and the available filters can be found in the next sections.
 
+## Tracee Kubernetes Policy configuration
+
+Through Querying the Tracee Configmap, users can access the default configuration and apply changes:
+```console
+kubectl edit ds/tracee -n tracee-system
```

Comment:
you talk about querying the ConfigMap but show how to edit a workload. (also s/querying/editing/)

Reply:
*"Tracee Configmap that is part of the Daemonset"

Reply:
I believe it will be a separate ConfigMap in the Operator